imagebuildah/stage_executor.go

```diff
@@ -1145,6 +1159,8 @@ func (s *StageExecutor) Execute(ctx context.Context, base string) (imgID string,
 		var (
 			commitName          string
 			cacheID             string
+			cacheKey            string
+			cachePulled         bool
```

Comment:
I’m tempted to suggest something like
```suggestion
			pulledAndUsedCacheImage         bool
```
— the larger a function, the more descriptive names should be (also, the value is not set when we just _pulled_ a cache image, but when `s.intermediateImageExists` matched it).

But that’s entirely up to the repo maintainers.